### DISCUSSION TITLE
Signature collection field on the screen.

### DISCUSSION THREAD

**Discussion by rogeriojpp** [Budibase/budibase#12409] [Help] [answered]
I'm developing a work order app and I would like to know if there is a signature collection field on the screen. Is this possible using Budibase?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @rogeriojpp,

You may be able to using a custom plugin to do exactly this. The plugin I believe you may want to use can be found [here](https://github.com/FlaminWrap/budibase-signature-component).

This [documentation](https://docs.budibase.com/docs/custom-plugin) is how you would go about installing the plugin. 

I hope this helps.
